### ISSUE TITLE
benchmarck closes in the process...

### ISSUE THREAD

**Issue by nandopais** [nicehash/NiceHashMiner-Archived#259]
hello I'm having trouble with "benchmarck" when I start it checks all the coins and the last he simply closes.
I have a core i3 3.20
4gb ddr3
1 x gygabite R9 380 4gb
1 x Sapphire R9 380 4gb
1 x powercolor r7 265 2gb
everything always worked perfectly in verssao 1.6.1.3, so I was moving to the 1.7.1.4 version that has given me such problems.
ok 
ok

**Comment by nandopais** [comment]
S74nk0
I performed the test and gave me all worked. I deactivated standard optimizations and managed to make benchmarck.
now it's all ok.
I would like to give an opnion. if possible create a protection, so that it turn off when the plates have very high temperatures, or even an rpm control for cooler
I did some tests yesterday and realized that the temperature of my r9 290x reached 95graus
luckily I was close because it could have burned.
I'm turning off amd temperature control with it enabled my cooler so reach 30%
I used the MSI Afterburner 100% rpm cooler to lower the temperature a little of vga however not help much
